web/LICENSE

```diff
@@ -1,203 +0,0 @@
-
```

Comment:
FYI, any artifact we distribute should have a clean `LICENSE` and `NOTICE`. I agree to remove for now, I will do a complete pass.

Reply:
We discussed on mailing list that there is plugin in parent pom that add these file during build, please check #410